tokio-stream/src/wrappers/broadcast.rs

```diff
@@ -0,0 +1,38 @@
+use crate::Stream;
+use async_stream::try_stream;
+use std::pin::Pin;
+use tokio::sync::broadcast::error::RecvError;
+use tokio::sync::broadcast::Receiver;
+
+use std::task::{Context, Poll};
+
+/// A wrapper around [`Receiver`] that implements [`Stream`]. Achieved by using the [`async-stream`] crate.
+///
+/// [`Receiver`]: struct@tokio::sync::broadcast::Receiver
+/// [`Stream`]: trait@crate::Stream
+/// [`async-stream`]: https://docs.rs/async-stream
+#[derive(Debug)]
+pub struct BroadcastStream<T: Clone> {
```

Comment:
I may be wrong, but I think we can take away this bound in the struct definition.
```suggestion
pub struct BroadcastStream<T> {
```

Reply:
Otherwise I see this:
```
error[E0599]: no method named `recv` found for struct `tokio::sync::broadcast::Receiver<T>` in the current scope
  --> tokio-stream/src/wrappers/broadcast.rs:33:26
   |
33 |                 match rx.recv().await {
   |                          ^^^^ method not found in `tokio::sync::broadcast::Receiver<T>`
   |
   = note: the method `recv` exists but the following trait bounds were not satisfied:
           `T: Clone`
```
So maybe it is needed?

Reply:
It's needed on the impl block, but not necessarily on the struct.

Reply:
Indeed. Didn't aware of that. Thank you.